templates/partner/intel.html

```diff
@@ -0,0 +1,645 @@
+            <h3 class="p-heading--5">
+              <a href="/blog/profile-workloads-on-x86-64-v3-to-enable-future-performance-gains">Profile workloads on x86-64-v3 to enable future performance gains
+              </a>
+            </h3>
+          </div>
+        </div>
+        <div class="p-equal-height-row__col">
+          <div class="p-equal-height-row__item u-hide--small u-hide--medium">
+            <div class="p-image-container">
+              {{ image(url="https://assets.ubuntu.com/v1/72db74d3-optimized-rt-ubuntu.png",
+                            alt="",
+                            width="852",
+                            height="1279",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-image-container__image"},) | safe
+              }}
+            </div>
+          </div>
+          <div class="p-equal-height-row__item">
+            <h3 class="p-heading--5">
+              <a href="/blog/optimised-real-time-ubuntu-is-now-generally-available-on-intel-socs">Optimized Real-time Ubuntu is now generally available on Intel SoCs</a>
+            </h3>
+          </div>
+        </div>
+        <div class="p-equal-height-row__col">
+          <div class="p-equal-height-row__item u-hide--small u-hide--medium">
+            <div class="p-image-container">
+              {{ image(url="https://assets.ubuntu.com/v1/cbd961da-performance-engineering.png",
+                            alt="",
+                            width="852",
+                            height="1279",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-image-container__image"},) | safe
+              }}
+            </div>
+          </div>
+          <div class="p-equal-height-row__item">
+            <h3 class="p-heading--5">
+              <a href="/blog/ubuntu-performance-engineering-with-frame-pointers-by-default">Performance engineering on Ubuntu leaps forward with frame pointers by default in Ubuntu 24.04 LTS</a>
+            </h3>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Accelerate data and AI applications with Ubuntu for Intel</h2>
+      </div>
+      <div class="col">
+        <p>
+          Canonical and Intel partner closely to ensure that data and AI applications can draw upon the most advanced hardware capabilities and accelerators in just a few commands. With Canonical's portfolio of data applications, AI tools, and infrastructure software, you'll be getting the most out of the hardware.
+        </p>
+        <div class="p-cta-block">
+          <!-- This needs to be updated after creation of new form -->
+          <a href="https:/canonical.com/contact-us" class="p-button--positive">Contact us</a>
```

Comment:
```suggestion
          <a href="https://canonical.com/contact-us" class="p-button--positive">Contact us</a>
```

Reply:
The comment says it has to be updated after creating a new form, is the new form supposed supposed to be created in this PR?

Reply:
Nope, in separate PR